app/src/pages/explorer/sample/sample.html

```diff
@@ -0,0 +1,57 @@
+<div class="sample-view">
+  <spinner v-if="loading" :loading="loading" :text="'Loading sample page'" />
+  <div v-else>
+    <div v-if="header" data-test="header">
+      <h1>{{ this.$route.params.label }}</h1>
+      <span>{{ header.sample_label }}</span>
+      <span>{{ header.process_label }}</span>
+      <div>DOI: {{ header.DOI }}</div>
+    </div>
+    <div v-if="materialComponents" data-test="materialComponents">
+      <h2>Material Components and Attributes</h2>
+      <ul v-for="material in materialComponents" :key="material.class">
+        <li>Class: {{ material.class }}</li>
+        <li>Role: {{ material.role }}</li>
+        <li v-for="property in material.materialProperties" :key="property.type">
+          <span>{{ property.type }}</span>:
+          <span>{{ property.value }}</span>
+          <span>{{ property.units }}</span>
+        </li>
+      </ul>
+    </div>
+    <div v-if="sampleImages" data-test="sampleImages">
+      <h2>Sample Images</h2>
+      <img v-for="image in sampleImages" :key="image.src" :src="image.src" :alt="image.alt" />

```

Comment:
I'll suggest using `:key="index"` instead of `:key="image.src"`.  Like so `v-for="(row, index) in sampleImages" :key="index`. Same for other areas where v-for is used except if the array comes with id field

Reply:
are you sure about giving the index as an id when there are multiple for loops in the sample page? React throws an error when the id is repeated in a component, I am not sure about Vue.